src/cuda/exec_dist.f90

```diff
@@ -56,24 +56,24 @@ subroutine exec_dist_tds_compact( &
   end subroutine exec_dist_tds_compact
 
   subroutine exec_dist_transeq_3fused( &
-    r_u, u, u_recv_s, u_recv_e, v, v_recv_s, v_recv_e, &
-    du, dud, d2u, &
+    r_du, u, u_recv_s, u_recv_e, v, v_recv_s, v_recv_e, &
+    dud, d2u, &
     du_send_s, du_send_e, du_recv_s, du_recv_e, &
     dud_send_s, dud_send_e, dud_recv_s, dud_recv_e, &
     d2u_send_s, d2u_send_e, d2u_recv_s, d2u_recv_e, &
     der1st, der2nd, nu, nproc, pprev, pnext, blocks, threads &
     )
     implicit none
 
-    ! r_u = -1/2*(v*d1(u) + d1(u*v)) + nu*d2(u)
-    real(dp), device, dimension(:, :, :), intent(out) :: r_u
+    ! r_du = -1/2*(v*d1(u) + d1(u*v)) + nu*d2(u)
```

Comment:
Is this comment still correct? If so, then I think `r_du` can be `intent(out)` as before?

Also, I think this comment can satisfy @Nanoseb's suggestion, perhaps explicitly drawing attention to this being the output here

Reply:
Yes, the comment is correct, fixed the `intent`.